src/main.rs

```diff
@@ -1,4 +1,5 @@
-use std::{collections::HashMap, error::Error, fs, io::Write, num::ParseIntError, ops::AddAssign};
+use loc::Loc;
+use std::{collections::HashMap, error::Error, fs, io::Write, ops::AddAssign};
 
```

Comment:
I have some commits which do not compile since I consider it more important to save progress more often rather than always ensuring the code to compile.  The latest version does compile.